specification/app/Microsoft.App.DynamicSessions/models/code-execution.tsp

```diff
@@ -0,0 +1,96 @@
+import "./common.tsp";
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Microsoft.App.DynamicSessions;
+
+@doc("The request to execute code.")
+model SessionCodeExecutionRequest {
+  @doc("The code input type of the code execution request.")
+  codeInputType: CodeInputType;
+
+  @doc("The execution type of the code execution request.")
+  executionType: ExecutionType;
+
+  @doc("The string of the code to execute, based on CodeInputType.")
+  code: string;
+
+  @doc("Code execution timeout in seconds.")
+  timeoutInSeconds: int64;
+}
+
+@doc("The response containing the result of the code execution.")
+model SessionCodeExecutionResponse {
+  ...SessionCodeExecutionRequest;
+
+  @visibility("read")
+  @doc("Session code execution id.")
+  id: string;
+
+  @doc("The status of the code execution, indicates whether succeeded or not.")
+  status: CodeExecutionStatus;
+
+  @doc("The standard output of the code execution.")
+  stdout: string;
+
+  @doc("The standard error of the code execution.")
+  stderr: string;
+
+  #suppress "@azure-tools/typespec-azure-core/no-unknown" ""
+  @doc("The result of the code execution.")
+  result: unknown;
```

Comment:
What exactly is this? A JSON object? You had to tell customer how to parse it?

Reply:
Sure, added more information in the doc.

Reply:
What do you mean by "after being Json serialized"? Could code execution return nothing or return some text without structure?

I couldn't find an meaningful "result" value in your examples.